### PR TITLE
Bug fixed: #349.

### PR DESCRIPTION
**Pull request by StoryHack**
Replaces 'Edit / Delete' link in admin navbar with a 'View All Episodes' link to a new page (admin/episodes_list.php) that shows all episodes, even the future ones, ordered by decending date, as most users will want to edit upcoming and recently published, so they shouldn't need to scroll.

* [x] I am the author of this code or the code is public domain
* [x] I release this code into the public domain

### CODE REVIEW

**Comment by StoryHack** [comment]
This is particularly bothersome, as if you change the publication date, then PG doesn't know how to save the new xml file and rename the mp3.

This is one of the dangers of storing the publication date as part of the filename, rather than as a field in the xml.

**Review by coldacid — APPROVED** [review]
I haven't tried this but it looks good after eyeballing the code.

**Review by emilengler — CHANGES_REQUESTED** [review]
This is useful thanks! However I think it's better to not re-invent the wheel extend the getEpisodes function with an argument to ignore the timestamp of files.

File: PodcastGenerator/admin/episodes_list.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+# This is Free Software released under the GNU/GPL License.
+############################################################
+require 'checkLogin.php';
+require '../core/include_admin.php';
+
+//$episodes = getEpisodes(null, $config, true);
+
+// reads all episodes into an array
+// and yes, I know this does basically what the other getEpisodes function does...
+function getEpisodeArray(){
```
Comment (coldacid) [inline PodcastGenerator/admin/episodes_list.php:17]: Per https://github.com/PodcastGenerator/PodcastGenerator/pull/374#pullrequestreview-579086708 could you update the original `getEpisodes()` function and make use of it here? Thanks.

File: PodcastGenerator/admin/episodes_list.php [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+$episodes = array_reverse($episodes);
+
+
+
+?>
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title><?php echo htmlspecialchars($config['podcast_title']); ?> - <?php echo _('Theme Buttons'); ?></title>
```
Comment (coldacid) [inline PodcastGenerator/admin/episodes_list.php:51]: Probably should not be "Theme Buttons"

**Review by coldacid — CHANGES_REQUESTED** [review]

**Pull request merged**